lib/mayaHydra/mayaPlugin/renderOverride.cpp

```diff
@@ -1,5 +1,5 @@
 //
-// Copyright 2019 Luma Pictures
```

Comment:
maybe double check with Sylvain, are we fine to change the copyright info here?

Reply:
Yep. I asked the same question on the side. I would think we may need to keep it. Will update when I more info.

Reply:
keeping it is the way to go.